apps/lcs/translations/src/en/validation.json

```diff
@@ -7,13 +10,13 @@
     "before": "This date must not be in the future"
   },
   "tenant-full-name": {
-    "required": "Enter the [X]'s full name"
+    "required": "Enter the {{valuesEnriched.tenantType}}'s full name"
   },
   "tenant-dob": {
-    "required": "Enter the [X]'s date of birth",
-    "test": "[X] cannot be born after they moved into the rental property. Enter a date after [Y].",
+    "required": "Enter the {{valuesEnriched.tenantType}}'s date of birth",
+    "test": "{{valuesEnriched.tenantType}} cannot be born after they moved into the rental property. Enter a date after {{valuesEnriched.tenantDoB}}.",
```

Comment:
is `tenantDoB` the right value to enter a date after? Should it not be more like `Enter a date before {{valuesEnriched.moveInDate}}.` (I don't pretend to understand the complexity of dates in this form but it seems a bit weird to me)